src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int upbothside;
+    private int downbothside;
+    private int height;
+
+    public IsoscelesTrapezoid(String color, int upbothside, int downbothside, int height) {
+        super(color);
+        this.upbothside = upbothside;
+        this.downbothside = downbothside;
+        this.height = height;
+    }
+
+    @Override
```

Comment:
The `draw()` method and the `getArea()` method should not be in the same class as they are conceptually different. Consider separating them into different interfaces or classes.